common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
But with @alisan617's change in #13293, msg will be wrapped with HTML where it is constructed. But it seems that there is no harm in also have HTML here.